### PR TITLE
Renamed packages and namespaces

### PR DESCRIPTION
**Pull request by colombod**
This pr is 
*  adopting pi-top as root name for nuget packages
*  renaming PiTopMakerArchitecture.* to PtTop.MakerArchitecture.*
*  adding first part of expansionPlate work

### CODE REVIEW

File: src/PiTop.nuget/PiTop.nuget.csproj [hunk 1]
```diff
@@ -5,17 +5,17 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <PackageVersion>0.0.1</PackageVersion>
-    <PackageId>piTop</PackageId>
+    <PackageId>pi-top</PackageId>
     <IsPackable>true</IsPackable>
-    <PackageDescription>piTop module.</PackageDescription>
+    <PackageDescription>pi-top board.</PackageDescription>
```
Comment (m-roberts) [inline src/PiTop.nuget/PiTop.nuget.csproj]: The device is a `pi-top [4]`, and it is not a board. Let's try and come up with a better description for this! 😄

Comment (colombod) [reply src/PiTop.nuget/PiTop.nuget.csproj]: Please let us know as we are almost ready for merge:)

File: src/PiTop.Camera/StreamingServer.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
```
Comment (m-roberts) [inline src/PiTop.Camera/StreamingServer.cs]: This file should be deleted

File: src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
```
Comment (m-roberts) [inline src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs]: These functions should move into the Algorithms section

**Comment by m-roberts** [comment]
Move rover.md closer to rover sample.
Review `libs/` - can we move them to NuGet? Or at least standardise the files in there.. include version numbers, source, build scripts, etc? Let's add 32 and 64 bit versions of each.
Need to open an issue with the .NET folks re: SMBus device - can we move this out of this repo?

**Review by m-roberts — CHANGES_REQUESTED** [review]
@colombod 😎

**Comment by m-roberts** [comment]
Create NuGet packages for native runtimes

File: src/PiTop.Camera/StreamingServer.cs (hunk 2 above)
Comment (colombod) [reply src/PiTop.Camera/StreamingServer.cs]: deleted

File: src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs (hunk 3 above)
Comment (colombod) [reply src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs]: moved and moved tests too

File: runtimes/Microsoft.Onnx.Runtime/1.5.2/pit-top.Onnx.runtime.nuspec [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>pit-top.Onnx.runtime</id>
+        <version>1.5.2</version>
+        <title>Onnx native bindings for pit-top OS</title>
```
Comment (m-roberts) [inline runtimes/Microsoft.Onnx.Runtime/1.5.2/pit-top.Onnx.runtime.nuspec]: `pit-top` -> `pi-top`

Comment (m-roberts) [reply runtimes/Microsoft.Onnx.Runtime/1.5.2/pit-top.Onnx.runtime.nuspec]: This occurs in a couple of places 😝

File: src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs (hunk 3 above)
Comment (colombod) [reply src/PiTop.MakerArchitecture.Expansion/MathHelpers.cs]: done

File: runtimes/Microsoft.Onnx.Runtime/1.5.2/pit-top.Onnx.runtime.nuspec (hunk 4 above)
Comment (colombod) [reply runtimes/Microsoft.Onnx.Runtime/1.5.2/pit-top.Onnx.runtime.nuspec]: because I am good

**Pull request merged**